pkg/inputs/snmp/x/meraki/meraki.go

```diff
@@ -301,16 +310,19 @@ type client struct {
 	appUsage         []appUsage
 }
 
-func (c *MerakiClient) getNetworkClients() ([]*kt.JCHF, error) {
+func (c *MerakiClient) getNetworkClients(dur time.Duration) ([]*kt.JCHF, error) {
 	clientSet := []*client{}
+	durs := float32(3600) // Look back 1 hour in seconds, get all devices using APs in this range.
```

Comment:
@i3149 - would this create duplicate data if the polling interval is less than 1 hour? 

curious why we wouldn't just use the `poll_time_sec` to navigate this one.

Reply:
Good call out! This was debugging code which slipped in. Updated.